test/resources/resources.yaml

```diff
@@ -72,3 +72,13 @@ metadata:
     findme: "yup"
 data:
   secured.txt.url: "http://dummy-server/secured"
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: url-configmap-binary-download
+  labels:
+    findme: "yup"
+binaryData:
+  # Base64 encoded url is 'https://github.com/kiwigrid/k8s-sidecar/raw/master/test/kubelogo.png'
```

Comment:
I'd have preferred to target the dummy server (written in python) for the test case as this adds a dependency to an external service to every test run. But since we're running on github and if github is down the run is somewhat useless anyway this would be ok for me.